pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -204,22 +211,23 @@ func (h *handler) getChartsToInstall() []*chart.Definition {
 			},
 			Enabled: func() bool {
 				toEnable := false
-				suc, err := h.deployment.Get(namespace.System, sucDeploymentName)
+				suc, err := h.deploymentCache.Get(namespace.System, sucDeploymentName)
```

Comment:
Just curious, why are we changing to cache here?

Reply:
Ah, missed that - `deployment` name was used for cache already so there is no _actual_ logic change here. New `deployment` name is for the controller.